src/gatsby-theme-landing-page/components/layout.js

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
```

Comment:
In a real-world site, this component would probably exist somewhere else and the file here would re-export it. We can move this later to better show best practices and make the starter API simpler